Python.md

```diff
@@ -40,7 +85,37 @@ result = connection.execute(sqlalchemy.text("""
            show_gravatar
       FROM "user"
      WHERE musicbrainz_id IN :musicbrainz_usernames
-"""), {
-    'musicbrainz_usernames': tuple(usernames),
-})
+""")
+result = connection.execute(query, {'musicbrainz_usernames': tuple(usernames)})
 ```
+
+It is preferred to assign the query to a separate variable and not include it inside the
+`execute` call.
```

Comment:
mostly because of formatting, and a little bit of comprehension. By the time you have

```python
def function()
    with db.engine.connect() as connection:
        connection.execute(sqlalchemy.text("""SELECT .. 
                                                FROM ...
                                               WHERE ...   
                                            """), {"args"})
```
both the query and args are aligned very far to the right, meaning that the chances of running over the line width is higher, and pep-8 indentation means that we have to keep them there. By putting these in an arg, we move the text left, which I think makes it more readable.

I also have a personal preference to break these lines up, because they have a lot going on. It's my opinion that it makes the queries and logic more readable.